py/visdom/server.py

```diff
@@ -1053,7 +1091,17 @@ def main(print_func=None):
 
     logging.getLogger().setLevel(logging_level)
 
-    start_server(port=FLAGS.port, env_path=FLAGS.env_path, readonly=FLAGS.readonly, print_func=print_func)
+    if FLAGS.with_login:
+        username = input("Please input your username: ")
+        password = input("Please input your password: ")
```

Comment:
As a better practice, please use the `getpass` library to pull the password, this way it isn't just displayed in plaintext in terminal